Doc/howto/descriptor.rst

```diff
@@ -1580,9 +1582,9 @@ Python:
     class Object:
         'Simulate how object.__new__() allocates memory for __slots__'
 
-        def __new__(cls, *args):
+        def __new__(cls, *args, **kwargs):
             'Emulate object_new() in Objects/typeobject.c'
-            inst = super().__new__(cls)
+            inst = super().__new__(cls, *args, **kwargs)
```

Comment:
Remove this edit.  It is a breaking change.   

```
Traceback (most recent call last):
  File "/Users/raymond/Documents/tmp20.py", line 84, in <module>
    h = H(10, 20)
  File "/Users/raymond/Documents/tmp20.py", line 51, in __new__
    inst = super().__new__(cls, *args, **kwargs)
TypeError: object.__new__() takes exactly one argument (the type to instantiate)

```

Reply:
Fixed.